ui/src/apiContext.ts

```diff
@@ -319,6 +319,26 @@ class FakeExportAPI {}
 
 class FakeUsersAPI {}
 
+const getAccessToken = () => {
+  let accessToken = "";
+  if (process.env.REACT_APP_GET_LOCAL_AUTH_TOKEN) {
+    // For local dev only, get the bearer token from the iframe url param
+    accessToken =
+      new URLSearchParams(window.location.href.split("?")[1]).get("token") ??
+      "";
+  } else {
+    // Check parent window's localStorage for auth token
+    // Use try/catch block to prevent UI from breaking in case of CORS error
```

Comment:
What is the purpose of this? Isn't the UI already broken if the auth token isn't available?

Reply:
I added this when we were still getting the domains sorted in AoU test to prevent a blank page on CORS error. Since the Workbench and SD Tanagra UIs will now be on the same domains as the parent window, should be fine to remove it.